plugins/identity/spec/controllers/projects_controller_spec.rb

```diff
@@ -182,45 +159,6 @@
       end
     end
 
-    context "masterdata_cockpit and networking services are not available" do
```

Comment:
it is ok to remove the whole spec? because it removes also tests for masterdata, networking, sharding

Reply:
You are right, that was too much. Now only the resource_management related test is removed, as originally intended.